index.html

```diff
@@ -0,0 +1,395 @@
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkakrqj" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <div class="name-form">
+            <input
+              id="name"
+              placeholder="First name"
+            />
+            <input
+              id="surname"
+              placeholder="Last name"
+            />
+          </div>
```

Comment:
- [ ] Nice job using the correct input tag with a descriptive placeholder text 👍 👍   but according to the project requirements, there are a few things I think you still need to improve on here. 😃 
- - According to the project requirements, a user should not be able to submit a form with an empty field. 👍 
> A user cannot submit the form if one or more of the fields is empty.
- - So I think you need to mark all input fields required, you can achieve this by adding the `required` attribute to the inputs. 🍒 😄 
>
- - Also according to the project requirements, All inputs tag should have the correct type attributes 👏 
> All fields have the correct type.
- - So I think you should consider adding the type attribute to your first name, and last name input.  you can do that by using `type="text"` 👍 

- [ ] Finally, the last name and the first name input field should not be visible in the mobile view according to the Figma design
- Figma design
![Screenshot 2023-03-16 235239](https://user-images.githubusercontent.com/107632780/225771652-6eea8817-1920-4ee1-ac79-c79bd13d0c5e.png)

- Your Design
![Screenshot 2023-03-16 235354](https://user-images.githubusercontent.com/107632780/225771713-881079b5-9dbb-40e1-8d55-a69591b5ecf5.png)

- Please consider using the CSS display properties to hide the inputs filed in your media query.